runtime/src/bank.rs

```diff
@@ -1595,23 +1602,28 @@ impl Bank {
             total_rewards,
             distributed_rewards,
             stake_rewards_by_partition,
+            parent_blockhash,
```

Comment:
Instead of passing this hash back up, we could duplicate the code from https://github.com/solana-labs/solana/pull/34744, ie:
```
self
    .parent()
   .expect("Partitioned rewards calculation must still have access to parent Bank.")
   .last_blockhash();
```
But I preferred doing that expect only once.

Reply:
Question, is `last_blockhash` deterministic across the cluster?

Reply:
I have to brush up on `last_blockhash`. It has been a while since I last looked at this code. 
last_blockhash is the hash of all the tick/transaction_entry in the block. It only gets updated at the block_boundary (i.e. reaching the maximum tick_height).
When we are playing current bank, the blockhash for the parent bank should have already been registered. And the return of last_blockhash() from parent bank is deterministic.